### PR TITLE
Improve visibility of models gallery

### PR DESCRIPTION
**Pull request by Bultako**
This small PR intends to improve the visibility of models gallery adding a section **Buit-in models** in the Modelling and also in the Tutorials index pages of the docs.

### CODE REVIEW

**Comment by adonath** [comment]
Thanks @Bultako! I assigned @luca-giunti as reviewer, as he proposed this change in the dev call...

**Comment by adonath** [comment]
@Bultako I think one could even put an additional link at the front-page in the "Getting started" section e.g. between "Tutorials" and "How To".

**Comment by Bultako** [comment]
> @Bultako I think one could even put an additional link at the front-page in the "Getting started" section e.g. between "Tutorials" and "How To".

Ah, you mean in the horizontal navigation menu.
Ok, done.

I have also renamed the Getting Started page into Start to not have the same name as the Getting Started group in the menu, and added a link in the Modeling and Fitting section of the Overview page.

File: docs/getting-started.rst [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 .. _getting-started:
 
-Getting Started
-===============
+Start
```
Comment (luca-giunti) [inline docs/getting-started.rst:5]: Maybe "Documentation", or "Quickstart Guide"?

File: docs/modeling/index.rst [hunk 2]
```diff
@@ -42,6 +42,13 @@ datasets and thus provide links, allowing e.g. to do a joint fit to multiple
 IACT datasets, or to a joint IACT and Fermi-LAT dataset. Many examples are given
 in the tutorials.
 
+Built-in models
+===============
+
+Gammapy provides a series of spatial, spectral and temporal models.
```
Comment (luca-giunti) [inline docs/modeling/index.rst:48]: Maybe series -> large choice ?

File: docs/overview.rst [hunk 3]
```diff
@@ -130,7 +130,9 @@ Modeling and Fitting
 Beyond `Dataset` objects, Gammapy provides numerous functionalities related
 to data modeling and fitting, as well as data simulation.
 This includes spectral, spatial and temporal model classes to describe the gamma-ray sky.
-Gammapy also contains a complete API for model parameter handling and model fitting.
+Gammapy also contains a complete API for model parameter handling and model fitting,
+with a series of spatial, spectral and temporal models. You may check out the whole list
```
Comment (luca-giunti) [inline docs/overview.rst:134]: Same comment as above (series->large choice)

File: docs/tutorials/index.rst [hunk 4]
```diff
@@ -101,6 +101,13 @@ Time
    analysis/time/light_curve_flare.ipynb
    analysis/time/pulsar_analysis.ipynb
 
+Built-in models
+---------------
+
+Gammapy provides a series of spatial, spectral and temporal models.
```
Comment (luca-giunti) [inline docs/tutorials/index.rst:107]: Same comment as above (series->large choice)

**Review by luca-giunti — APPROVED** [review]
Thanks @Bultako! Very nice

**Comment by luca-giunti** [comment]
Please @adonath merge it, if you also approve. Thanks!

**Review by adonath — APPROVED** [review]
Thanks @Bultako! No further comment from my side...

**Pull request merged**